src/Illuminate/Cache/DatabaseStore.php

```diff
@@ -133,11 +158,29 @@ public function get($key)
      */
     public function put($key, $value, $seconds)
     {
-        $key = $this->prefix.$key;
-        $value = $this->serialize($value);
+        return $this->putMany([$key => $value], $seconds);
+    }
+
+    /**
+     * Store multiple items in the cache for a given number of seconds.
+     *
+     * @param  int  $seconds
+     * @return bool
+     */
+    public function putMany(array $values, $seconds)
```

Comment:
analog pls.

Reply:
/**
     * Store multiple items in the cache for a given number of seconds.
     */
    public function putMany(array $values, int $seconds): bool

Reply:
Again, I'd rather keep the same method signature as the interface we're implementing. Plus, the type of the $seconds variable cannot be enforced because of the interface definition.